nats-streaming-server.go

```diff
@@ -195,6 +198,9 @@ func parseFlags() (*stand.Options, *natsd.Options) {
 	flag.Int64("file_fds_limit", stores.DefaultFileStoreOptions.FileDescriptorsLimit, "FileStoreOpts.FileDescriptorsLimit")
 	flag.Int("io_batch_size", stand.DefaultIOBatchSize, "IOBatchSize")
 	flag.Int64("io_sleep_time", stand.DefaultIOSleepTime, "IOSleepTime")
+	flag.String("ft_group", "", "FTGroupName")
+	flag.Int("ft_quorum", 1, "FTQuorum")
```

Comment:
I started with group size but then you have to do the math to know that with (our) raft, quorum will be `group_size/2+1`. So I took the opposite approach where the user specifies the quorum, which is the number of servers required for the election. They can be running more servers if they want.